libraries/eventformatter/impl/src/main/kotlin/io/element/android/libraries/eventformatter/impl/ProfileChangeContentFormatter.kt

```diff
@@ -24,6 +25,7 @@ class ProfileChangeContentFormatter @Inject constructor(
     private val sp: StringProvider,
 ) {
     fun format(
+        senderId: UserId,
```

Comment:
Also, now that we pass the `senderId`, the parameter `senderIsYou` may be removed? The MatrixClient will have to be injected in the constructor. I understand that this is maybe temporary code, so this remark can be ignored I guess.